pywinauto/win32structures.py

```diff
@@ -153,13 +153,9 @@ def _reduce(self):
 WPARAM = wintypes.WPARAM
 
 
-class POINT(Structure):
-    _pack_ = 4
-    _fields_ = [
-        # C:/PROGRA~1/MIAF9D~1/VC98/Include/windef.h 307
-        ('x', LONG),
-        ('y', LONG),
-    ]
+class POINT(wintypes.POINT, Structure):

```

Comment:
Is `wintypes.POINT` not enough? It's not clear why multiple inheritance is needed here.

Reply:
@vasily-v-ryabov: The unit tests failed (this is why I added this in my 3rd commit). By default `ctypes.Structure` descendents don't know how to compare their *\_fields\_* content to a tuple, that's what `Structure.__eq__` does. I wanted to use the mixin technique, but apparently I introduced a nice diamond of hell. As soon as tests finish, I'll correct the issue (replacing it by a workaround).

Secondly, I don't know why I get coverage reports stating a drop from 90+% to 60+%.

Reply:
Regarding the coverage report, I think, it's about changes of signatures for functions that are never called.

Reply:
Coverage report starts from 60% because every Python version submits coverage and the bot posts incomplete coverage first, then it edits the post with every update after every line of AppVeyor matrix done.

Regarding commit email you can see that all commits are gray and not bound to your GitHub account. So your contribution won't be counted correctly here: https://github.com/pywinauto/pywinauto/graphs/contributors

The reason of the gray commits that you used Git local settings for "username" and "email" different than in your GitHub account. See https://help.github.com/en/articles/setting-your-commit-email-address-in-git

Reply:
@vasily-v-ryabov: Thank you for the details.

Regarding the username, I figured that out in the meantime and I changed it (by default it was empty, and my PC username was used instead), but apparently for pushed commits it's not a trivial task. So I don't mind if my username won;t appear in the contributors list. My goal is having this fixed, not going into the hall of fame.

Reply:
No problem. I can fix this if you send me an email to `"vasily-v-ryabov".replace('-', '.')` at gmail(dot)com. `git rebase` command is needed. Maybe this script is useful: https://help.github.com/en/articles/changing-author-info (but I'd like to check this carefully).

Reply:
One more useful answer: https://stackoverflow.com/a/1320317/3648361